app/src/main/java/com/orange/ods/app/ui/components/imagetile/ComponentImageTile.kt

```diff
@@ -88,41 +92,68 @@ fun ComponentImageItem() {
+                    OdsImageTileCaptionDisplayType.Overlay,
+                    OdsImageTileCaptionDisplayType.None -> imageSize
                 }
-                OdsImageItem(
-                    image = rememberAsyncImagePainter(
-                        model = recipe.imageUrl,
-                        placeholder = painterResource(id = R.drawable.placeholder),
-                        error = painterResource(id = R.drawable.placeholder)
-                    ),
-                    uncheckedIcon = if (hasIcon) painterResource(id = R.drawable.ic_heart_outlined) else null,
-                    checkedIcon = if (hasIcon) painterResource(id = R.drawable.ic_heart) else null,
-                    title = if (hasText) recipe.title else null,
+                OdsImageTile(
                     modifier = Modifier
                         .width(imageSize)
                         .height(height),
+                    onClick = { clickOnElement(context, context.getString(R.string.component_image_tile)) },
+                    image = OdsImageTileImage(
+                        rememberAsyncImagePainter(
+                            model = recipe.imageUrl,
+                            placeholder = painterResource(id = R.drawable.placeholder),
+                            error = painterResource(id = R.drawable.placeholder)
+                        ), ""
+                    ),
+                    uncheckedIcon = if (hasIcon) {
+                        OdsIconButtonIcon(
+                            painterResource(id = R.drawable.ic_heart_outlined),
+                            stringResource(id = R.string.component_button_icon_toggle_favorite_add_icon_desc)
+                        )
+                    } else null,
+                    checkedIcon = if (hasIcon) {
+                        OdsIconButtonIcon(
+                            painterResource(id = R.drawable.ic_heart),
+                            stringResource(id = R.string.component_button_icon_toggle_favorite_remove_icon_desc)
+                        )
+                    } else null,
+                    title = if (hasText) recipe.title else null,
                     iconChecked = iconChecked,
-                    iconContentDescription = stringResource(id = R.string.component_button_icon_toggle_favorite_add_icon_desc),
                     onIconCheckedChange = { checked -> iconChecked = checked },
-                    displayTitle = if (isOverlay) OdsImageItemTitleType.Overlay else if (isBelow) OdsImageItemTitleType.Below else OdsImageItemTitleType.None,
+                    captionDisplayType = when {
+                        isOverlay -> OdsImageTileCaptionDisplayType.Overlay
+                        isBelow -> OdsImageTileCaptionDisplayType.Below
+                        else -> OdsImageTileCaptionDisplayType.None
+                    },
```

Comment:
```suggestion
                    captionDisplayType = type.value,
```

`isOverlay` and `isBelow` in `ImageTileCustomizationState` can also be removed.